pkgs/development/tools/hobbes/default.nix

```diff
@@ -1,29 +1,30 @@
-{ lib, stdenv, fetchFromGitHub, cmake, llvm_6, ncurses, readline, zlib, libxml2 }:
+{ lib, stdenv, fetchFromGitHub, cmake, llvm_12, ncurses, python27, readline, zlib, libxml2 }:
```

Comment:
so, `python27` is needed for the tests, but it seems that it becomes a dependency... does this mean that a _user_ is going to have to have python? or do we skip checks completely for releases?

Reply:
It is only for tests but we no longer allow new usages of python27. Either get it to work with python3 or disable the tests.

Reply:
ack; will do